contracts/optimistic-ethereum/OVM/precompiles/OVM_SequencerEntrypoint.sol

```diff
@@ -23,7 +23,7 @@ contract OVM_SequencerEntrypoint {
     /*********
      * Enums *
      *********/
-    
```

Comment:
These whitespace diffs add noise to code review. We need linter rules that prevent such things from getting into the codebase